src/Layout/LandingPage/index.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Redirect } from 'react-router-dom'
+import { Title, LandingImg, Paragraph, Button, ResourceLink, Link } from './index.style';
+import landing from '../../assets/landing-img.jpeg';
+import Header from '../../CommonComponents/Header';
+
+class LandingPage extends React.Component {
+
+  state = {
+    redirectBtn: false,
+    renderLink: false
+  }
+
+  setRedirect = () => {
+    this.setState({
+      redirectBtn: true
+    })
+  }
+
+  renderBtn = () => {
+    if (this.state.redirectBtn) {
+      return <Redirect to='/scheme' />
+    }
+  }
+
+
+
+  setRender = () => {
+    this.setState({
+      renderLink: true
+    })
+  }
+
+  renderLink = () => {
+    if (this.state.renderLink) {
+      return <Redirect to='/resource' />
```

Comment:
The endpoint name should be resources not resource

Reply:
@95Rawan 
I Fixed it!